const number = 0;

var minhaVar = 'Oi';

console.log(minhaVar);



// try {
//     throw new EvalError("Hello");
//   } catch (e) {
//     console.log(e instanceof EvalError); // true
//     console.log(e.message); // "Hello"
//     console.log(e.name); // "EvalError"
//     console.log(e.stack); // Stack of the error
//   }